presidio-analyzer/analyzer/predefined_recognizers/ip_recognizer.py

```diff
@@ -1,6 +1,7 @@
 from analyzer import Pattern
 from analyzer import PatternRecognizer
 
+# pylint: disable=line-too-long,abstract-method
```

Comment:
shouldn't have abstract-method lint. @abstractmethod should be removed from the validate_result function in PatternRecognizer

Reply:
This guy (@AbstractMethod) gets a reminder every time someone writes @AbstractMethod in a comment...